transport/http/src/main/java/org/kaazing/gateway/transport/http/HttpAcceptor.java

```diff
@@ -268,6 +276,8 @@ void bindInternal(final ResourceAddress address, IoHandler handler,
         //
         final ResourceAddress transportAddress = address.getTransport();
 
+        handshakeTimeoutByTransportAddress.put(transportAddress, address.getOption(HANDSHAKE_TIMEOUT).longValue());
```

Comment:
Does this mean that if a certain resource address already exists in the map, then the smallest value should be kept ? 
Also, if there is a case when the user set value is higher than the default value (of 10 seconds), which value should be kept ?